wrappers/azurekeyvault/azurekeyvault.go

```diff
@@ -287,6 +289,8 @@ func (v *Wrapper) buildBaseURL() string {
 	return fmt.Sprintf("https://%s.%s/", v.vaultName, v.environment.KeyVaultDNSSuffix)
 }
 
+var managedClientIdLock sync.Mutex
```

Comment:
Can this be moved into the Wrapper struct?

Reply:
No, it has to be global, as the race is between more than one instance of Wrapper being initialized in Vault, when used with Seal High Availability.

Reply:
Ty!